gatsby/package.json

```diff
@@ -26,7 +26,9 @@
     "lint:ts": "eslint \"**/*.{js,jsx,ts,tsx}\"",
     "lint:ts:fix": "npm run lint:ts -- --fix",
     "// TYPECHECKING": "",
-    "typecheck": "tsc --noEmit"
+    "typecheck": "tsc --noEmit",
+    "//Teamcity": "",
+    "audit:teamcity": "npm audit --json|npm-audit-reporter-teamcity"
```

Comment:
Can you add  `--audit-level=moderate` in here please? This should cause a non-0 exit code for moderate or above issues